community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -251,6 +253,11 @@ public void doImport( Input input ) throws IOException
                 nodeLabelsCache.close();
             }
         }
+
+        if ( success )
+        {
+            log.info( "Import completed successfully, took " + Format.duration( totalTimeMillis ) + ". " + storeUpdateMonitor );
+        }
```

Comment:
I don't think we need the success variable, since the `catch ( Throwable t )` above will throw a laundered exception on failure, and thus avoiding this code path.